### PR TITLE
BL-464-fe-i-want-to-be-able-to-express-my-interest-in-the-program

### PR DESCRIPTION
**Pull request by techlando**
## Description

Created Feature in JoinUpdate.js file, where we stored simple form for people who are not authenticated on landing page above footer. Had to make a axios call using a environmental variable for API access on the header. During development stage, we input component  above the footer, however we could not get UseOktaAuth working properly to authenticate if a user is signed in or not. The form is complete and ready for implementation.

Fixes # (issue) "FE: I want to be able to express my interest in the program"

## Loom Video
https://loom.com/share/ae1bdd7726cf4f588b75953595f3646f

## Type of change

Please delete options that are not relevant.

- [* ] New feature (non-breaking change which adds functionality)
- [ *] This change requires a documentation update

## Checklist:

- [ *] My code follows the style guidelines of this project
- [ *] I have performed a self-review of my own code
- [ *] I have removed unnecessary comments/console logs from my code
- [ *] My changes generate no new warnings
- [* ] I have checked my code and corrected any misspellings
- [ *] No duplicate code left within changed files
- [* ] Size of pull request kept to a minimum
- [ *] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by techlando** [comment]
authorization to check if a user is logged in or not still needs to be added, however form is ready to be implemented. Form was removed from rendering above footer, until authorization is complete.

**Review by Michael-Subbarao — COMMENTED** [review]
The form looks good and loom video provides good context, however in the future you should only git add the files necessary for the ticket so you don't cause merge conflicts. The package.json and git ignore changes probably shouldn't be a part of this ticket.

**Comment by laurensruiz** [comment]
The form looks great! I like the styling done for it, as a user it is easy to understand. I would definitely break apart the tasks since a lot of work was done for this particular ticket.

**Review by laurensruiz — APPROVED** [review]
The form looks great! I like the styling done for it, as a user it is easy to understand. I would definitely break apart the tasks since a lot of work was done for this particular ticket.

File: src/components/common/JoinUpdate.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+  const onChange = event => {
+    const name = event.target.name;
+    setValues({ ...value, [name]: event.target.value });
+  };
+  const onSubmit = e => {
+    e.preventDefault();
+    axios
+      .post(
+        'https://api.airtable.com/v0/app4rHdRcowslAwKp/Table%201',
```
Comment (paulstgermain) [inline src/components/common/JoinUpdate.js:23]: This is all looking great, just one question: Why are we pushing this data to Airtable? Who's Airtable is this going to?

Comment (techlando) [reply src/components/common/JoinUpdate.js:23]: Not sure of why airtable... but This is going to John's airtable, he stated he would have to get with Brianna and make her an account.

**Review by AnthonyLayne — APPROVED** [review]
This looks great, your code is very clean and easy to understand. I also appreciate the comments at the bottom, but maybe a comment explaining which one should be used once the site is launched. Other than that it looks good.

File: src/components/common/JoinUpdate.js (hunk 1 above)
Comment (jpivvt) [reply src/components/common/JoinUpdate.js:23]: This is an Airtable created by Brianne, as it was her preferred location to receive requests for information over the use of sending forms to email. I actually realized later that I only had an API key I created for this function. I'll have to make another task to change the key to hers, once I'm able to retrieve that from her.

**Review by EmmS21 — APPROVED** [review]
Lovely work. What are you using the Airtable link for, I didn't quite get that?

**Comment by techlando** [comment]
That's where the data is being stored. It was in the ticket to use airtable 🤷🏼‍♂️

**Comment by jpivvt** [comment]
> That's where the data is being stored. It was in the ticket to use airtable 🤷🏼‍♂️

Right. That's where the stakeholder had requested we send responses to the form, instead of using email to deliver them.

**Pull request merged**